---
title: |
  Only You:
  A Field Experiment of Text Message to Prevent Free-riding in Japan Marrow Donor Program
author:
  - name: Hiroki Kato
    affiliations:
      - Graduate School of Economics, Osaka University, Japan
  - name: Fumio Ohtake
    affiliations:
      - Graduate School of Economics, Osaka University, Japan
      - Center for Infectious Disease Education and Research (CiDER), Osaka University, Japan
  - name: Saiko Kurosawa
    affiliations:
      - Department of Oncology, Ina Central Hospital, Japan
  - name: Kazuhiro Yoshiuchi
    affiliations:
      - Graduate School of Medicine, Tokyo University, Japan
  - name: Takahiro Fukuda
    affiliations:
      - Department of Hematopoietic Stem Cell Transplantation, National Cancer Center Hospital, Japan
pdf-engine: lualatex
format:
  pdf:
    documentclass: article
    papersize: a4paper
    fig-format: pdf
    toc: false
    number-sections: true
    keep-tex: true
    include-in-header:
      - tex/header.tex
latex-output-dir: docs/PDF
execute:
  echo: false
  warning: false
---

In [ ]:
#| include: false

library(here)
library(IRdisplay)
source(here("R", "_library.r"))
source(here("R", "_outcome_labels.r"))

root <- "D:/JMDPフィールド実験"

# Field Experiment
## Background: Coordination Process of JMDP

骨髄バンクに登録したドナー候補者が幹細胞を提供するために、以下の工程がある

1. 適合通知の返信
    - 骨髄バンクに登録した患者のHLAが一致すると、骨髄バンクはその候補者に幹細胞提供を依頼する適合通知を受け取る
    - 適合通知を受け取ったドナー候補者は提供の意向を示して返信する
1. 確認検査
    - 1カ月以内に実施される
    - コーディネーターが提供方法を詳細に説明をし、ドナーの意思や家族の同意について調査する
    - 調整医師が問診、診察、感染症の有無や血液型を調べる一般血液検査を実施する
    - 骨髄バンクが設定している基準を満たしているかどうかを検査する
1. 第一候補者選定
    - 患者は同時に最大10人のドナーとのコーディネーションを進められる
    - 患者の主治医はその中から最もドナーに適した候補者を選ぶ
    - 重要なことは、ドナー候補者は患者が何人のドナーとのコーディネーションを進めているかを知ることができない。また、コーディネーターや調整医師も知らないので、ドナー候補者はその情報を得られない。
1. 最終同意
    - 第一候補者に選定されたドナーは、コーディネーターと調整医師からの説明を再度受けて、最終的な意思決定をする
    - ドナーは自身の意思だけでなく、家族の代表者の同意も必要である
    - 最終同意後、ドナーは自身の意向を変えられない
1. 採取
    - 最終同意後、術前検査と採取準備（貧血を防ぐための自己血採血）を実施する
    - 採取のために1週間程度の入院が必要である
    - 確認検査から採取まで3～4カ月程度かかる

## Experimental Design

- ドナー候補者確定後、骨髄バンクは対象者に幹細胞提供を依頼する「適合通知」と、それを郵送した旨を伝えるSNSメッセージを送付する。
- 適合通知の全文
  - この度、あなたと骨髄バンクの登録患者さんのHLA型（白血球の型）が一致し、ドナー候補者のおひとりに選ばれました。今後、ご提供に向け詳しい検査や面談を希望されるかをお伺いしたく連絡させていただきました。同封の資料をよくお読みいただき、コーディネートが可能かどうか検討の上、この案内が届いてから7日以内に返信用紙ほかをご返送ください。返送後、コーディネートを進めさせていただく場合は、担当者よりご相談のお電話を差し上げますのでよろしくお願い申し上げます。
- 我々の介入は行動科学の知見に基づくメッセージを適合通知に加える。
  - 確率メッセージ「1人の登録患者さんとHLA型が一致するドナー登録者は数百〜数万人に1人です。ドナー候補者が複数みつかる場合もありますが、多くはないこともご理解頂ければ幸いです。」
  - 患者情報メッセージ「骨髄バンクを介して移植ができる患者さんは約6割です。骨髄等を提供するドナーが早く見つかれば、その比率を高めることができます。」
- これらのメッセージの目的はコーディネーションを促進することである。ただし、我々はドナー候補者に過度なプレッシャーを与えないように適切な配慮をして、メッセージを作成した。
  - 嘆願調のようなメッセージを避けている
  - メッセージの作成に際して、骨髄バンクが公開している情報のみを使用している
  - 移植リスクに関する説明はこれまでと同様の方法で実施している
- 二つの介入メッセージの効果を推定するために、我々は3つの実験群を設ける。
  - 実験群A：介入メッセージなし
  - 実験群B：確率メッセージ
  - 実験群C：患者情報メッセージ
- さらに、情報過多による認知負荷の負の影響を検証するために、二つの介入メッセージを両方加えた適合通知を送付する実験群Dも設ける。

In [ ]:
#| include: false

schedule <- read_csv(here(root, "RCT-schedule.csv"))

In [49]:
#| output: asis

schedule %>%
  mutate(
    my = paste0(month, "/", year - 2000),
    my = factor(
      my,
      levels = c("9/21", "10/21", "11/21", "12/21", "1/22", "2/22"),
      labels = c("Sep 21", "Oct 21", "Nov 21", "Dec 21", "Jan 22", "Feb 22")
    ),
    week = factor(week, levels = 1:4, labels = paste("Week", 1:4))
  ) %>%
  dplyr::select(week, my, treat) %>%
  pivot_wider(values_from = treat, names_from = my) %>%
  kable(
    caption = 'Assignment Schedule',
    align = 'lcccccc',
    format = 'html',
    booktabs = TRUE,
    linesep = ""
  ) %>%
  kableExtra::kable_styling() %>%
  # print()
  as.character() %>%
  display_html()

Assignment Schedule 
 
 
 week 
 Sep 21 
 Oct 21 
 Nov 21 
 Dec 21 
 Jan 22 
 Feb 22 
 
 
 
 
 Week 1 
 B 
 C 
 C 
 D 
 B 
 A 
 
 
 Week 2 
 D 
 B 
 A 
 A 
 C 
 B 
 
 
 Week 3 
 A 
 D 
 B 
 C 
 D 
 C 
 
 
 Week 4 
 C 
 A 
 D 
 B 
 A 
 D

- 我々は2021年9月から2022年2月にかけて骨髄バンクが適合通知を送付したドナー候補者11,154名をフィールド実験の対象とした。
- 実験群の割り当ては骨髄バンク事務局の業務の無理のない範囲で週単位でクラスターランダム化した。
  - 週・月の固定効果を取り除けるように、実験群が週・月でバランスするように割り当てた。
  - 割り当てのスケジュールは上表にまとめている

## Data and Empirical Strategy

In [ ]:
#| include: false

rawdt <- read_csv(here(root, "shaped.csv"), locale = locale(encoding = "cp932"))

use <- rawdt %>%
  dplyr::filter(ongoing == 0) %>%
  mutate(
    treat = factor(treat, levels = LETTERS[1:4]),
    plan_two_methods = if_else(plan_method == "BM/PB", 1, 0)
  )

In [29]:
#| include: false

lm_robust(
  ongoing ~ treat,
  data = rawdt,
  cluster = RCTweek,
  se_type = "stata"
) %>%
{
  f <- summary(.)$fstatistic[1]
  numdf <- summary(.)$fstatistic[2]
  dendf <- summary(.)$fstatistic[3]
  p <- pf(f, numdf, dendf, lower.tail = FALSE)

  sprintf("F-value = %1.3f; p-value = %1.3f", f, p)
}

[1] "F-value = 0.105; p-value = 0.956"

- 我々は2022年6月末に骨髄バンクが管理するコーディネーションデータの提供を受けた。
  - 観測単位はフィールド実験の対象であるドナー候補者である
  - 個人属性として性別・年齢・過去のコーディネーション回数・居住地域（都道府県レベル）を記録している
  - コーディネーションの過程について、提供に至るまでの各工程に到達したかどうかを記録しており、これらをアウトカム変数として用いる
  - 適合通知の返信については、返信したかどうかに加えて、返信日数・提供意向についても記録きろくしている
  - コーディネーションが途中で中断した場合、その理由を三つのカテゴリー（患者理由・ドナーの健康以外の理由・ドナーの健康上の理由）で記録している
- 追加的なデータとして、JMDPがホームページ上で公開している施設リストを用いる
  - このデータは病院の住所に加えて、骨髄採取が可能かどうか、末梢血幹細胞採取が可能かどうかを含んでいる
  - 我々はこのデータを都道府県レベルで集計して、10平方キロメートル当たりの病院の数を計算し、コーディネーションデータと都道府県をキーとして突合する
  - この変数をコーディネーションや提供の移動コストとみなす
- 分析対象は分析対象は国内在住でコーディネーションが完全に終了している11,049名とする。
  - 海外に在住する人が1名いた
  - 現在もコーディネーションが進行している人が104名いた
  - コーディネーションの進行中の比率は実験群でバランスしている（F-value, p-value = $0.956$）

In [47]:
#| output: asis

balance_test <- use %>%
  select(
    male,
    age,
    coordinate,
    plan_two_methods,
    hospital_per_area,
    PB_per_area,
    BM_per_area,
    treat,
    RCTweek
  ) %>%
  pivot_longer(male:BM_per_area, values_to = "value", names_to = "vars") %>%
  group_by(vars) %>%
  do(est = lm_robust(
    value ~ treat,
    clusters = RCTweek,
    se_type = "stata",
    data = .
  )) %>%
  summarize(
    vars = vars,
    f = summary(est)$fstatistic[1],
    numdf = summary(est)$fstatistic[2],
    dendf = summary(est)$fstatistic[3],
    "F-test, p-value" = pf(f, numdf, dendf, lower.tail = FALSE)
  )

size <- with(use, sprintf("%1d", table(treat))) %>%
  {
    tribble(
      ~terms, ~A, ~B, ~C, ~D, ~"F-test, p-value",
      "通常の適合通知", "X", "X", "X", "X", "",
      "確率メッセージ", "", "X", "", "X", "",
      "患者情報メッセージ", "", "", "X", "X", "",
      "サンプルサイズ", .[1], .[2], .[3], .[4], ""
    )
  }

attr(size, "position") <- seq(nrow(size))

use %>%
  datasummary(
    (`Male (=1)` = male) +
    (`Age` = age) +
    (`Number of past coordinations` = coordinate) +
    (`Planned harvest method: BM and PBMC` = plan_two_methods) +
    (`Number of listed hospitals` = hospital_per_area) +
    (`Number of hospitals listed with PBSC collection` = PB_per_area) +
    (`Number of hospitals listed with BM collection` = BM_per_area) ~ mean * treat,
    data = .,
    title = "Overview of Field Experiment",
    add_rows = size,
    add_columns = balance_test[c(5, 1, 3, 7, 4, 2, 6), 5],
    align = "lccccc",
    output = "html"
  ) %>%
  kableExtra::kable_styling() %>%
  add_header_above(c(" " = 1, "Experimental Arms" = 4, " " = 1)) %>%
  group_rows("A. Interventions", 1, 3) %>%
  group_rows("B. Sample Size", 4, 4) %>%
  group_rows("C. Covariates, Balance Test", 5, 11) %>%
  # print()
  as.character() %>%
  display_html()

Overview of Field Experiment 
 
 
 
 Experimental Arms 
 
 
 
 
 A 
 B 
 C 
 D 
 F-test, p-value 
 
 
 
 A. Interventions 
 
 通常の適合通知 
 X 
 X 
 X 
 X 
 
 
 
 確率メッセージ 
 
 X 
 
 X 
 
 
 
 患者情報メッセージ 
 
 
 X 
 X 
 
 
 B. Sample Size 
 
 サンプルサイズ 
 2535 
 3053 
 2726 
 2735 
 
 
 C. Covariates, Balance Test 
 
 Male (=1) 
 0.62 
 0.63 
 0.63 
 0.61 
 0.36 
 
 
 Age 
 38.38 
 38.12 
 37.45 
 37.98 
 0.07 
 
 
 Number of past coordinations 
 1.61 
 1.59 
 1.62 
 1.56 
 0.45 
 
 
 Planned harvest method: BM and PBMC 
 0.87 
 0.89 
 0.87 
 0.90 
 0.28 
 
 
 Number of listed hospitals 
 0.48 
 0.49 
 0.49 
 0.49 
 0.80 
 
 
 Number of hospitals listed with PBSC collection 
 0.16 
 0.17 
 0.17 
 0.16 
 0.67 
 
 
 Number of hospitals listed with BM collection 
 0.25 
 0.26 
 0.25 
 0.25 
 0.80

- 上表はフィールド実験を概観している
  - パネルAは各実験群の介入をまとめており、パネルBは各実験群のサンプルサイズを示している
  - パネルCは共変量のバランステストの結果を示しており、割り当てのランダム化が成功しているかどうかを検証している
  - ほとんどの共変量は群間でバランスしているが、ドナー候補者の年齢は群間でアンバランスである可能性がある（F-test, p-value = $0.07$）
  - 実験群C・Dのドナー候補者は実験群A・Bのドナー候補者よりも若い
- ドナー候補者は実験群を選択できない、すなわち、実験群は外生的であるので、単純な二群比較は平均処置効果を識別できる。
  - 一部の共変量、および割り当ての週と月が実験群間で完全にバランスしていないので、単純な二群比較はバイアスを伴う可能性がある
  - そこで、我々は以下の線形確率モデルを推定する（$X_i$は個人属性ベクトル、$\lambda_m$と$\theta_w$はそれぞれ週・月のダミー変数）

\begin{equation}
  Y_{imw} =
  \beta_1 \cdot \text{B}_{mw} + \beta_2 \cdot \text{C}_{mw}
  + \beta_3 \cdot \text{D}_{mw}
  + X'_i \gamma + \lambda_m + \theta_w + u_{imw}
\end{equation}